### ISSUE TITLE
`attention_mask` does not cover entire padding region

### ISSUE THREAD

**Issue by ZhiyuanChen** [Shujun-He/RibonanzaNet#4]
The `attention_mask` is calculated in https://github.com/Shujun-He/RibonanzaNet/blob/efebd44e79615fb23f685971875e8903b04cbdce/Network.py#L347-L348

Since the masked position is marked as -1, the expanded attention will not be correctly calculated in the dot product. 

For instance, if a sequence has a length of 3 and padded into 5, we have: 

```python
[1, 1, 1, -1, -1]
```

The extended attention mask after dot product would be:

```python
[[ 1,   1,   1, -1, -1],
 [ 1,   1,   1, -1, -1],
 [ 1,   1,   1, -1, -1],
 [-1, -1, -1,  1,   1],
 [-1, -1, -1,  1,   1]]
```

While the expected extended attention mask should be:

```python
[[ 1,   1,   1, -1, -1],
 [ 1,   1,   1, -1, -1],
 [ 1,   1,   1, -1, -1],
 [-1, -1, -1, -1,  -1],
 [-1, -1, -1, -1,  -1]]
```
